README.adoc

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
```

Comment:
This should be the copyright claim asserted in all files.

Reply:
Thanks for the comment John.   Just want to confirm:  should this replace SmartBear's current copyright claim in the annotation interfaces?

Reply:
No, we shouldn't remove existing copyrights from other projects.  We need to recognize the original source of the code.  We're just adding to it with our copyright.

Reply:
We've processed an ECA for this right?

Reply:
from the email discussions, this will be in addition to the current copyright claim

Reply:
@johnament -  yup, I signed the ECA, and the code borrowed from SmartBear is under Apache v2 license.

Reply:
Then no, we didn't receive an ECA for this...